### DISCUSSION TITLE
Warning `CS0472` The result of the expression is always 'false'

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13103] [Q&A]
Following condition is always `false`

https://github.com/AvaloniaUI/Avalonia/blob/631a0eda452328b32ca3eea69949192e8e23c96d/src/Avalonia.X11/Screens/X11Screens.Scaling.cs#L200

Can i move the  condition to line `158` and remove `&& usePhysicalDpi == null` or remove line `200`?

Also, can I enable the analyzer to generate errors on the CS0472 warning?

**Comment by robloo** [comment]
This code might be leftover from before nullable ref types was enabled? But you are absolutely right that `usePhysicalDpi` is a standard bool so this is clearly a bug. It can't be null. I think @kekekeks knows the most about Linux backends.

> Also, can I enable the analyzer to generate errors on the CS0472 warning?

To me that seems like a good one to enable (and I know I haven't said that about others)
